src/components/HeaderContainer/HeaderContainer.js

```diff
@@ -6,28 +6,13 @@ import MovieCard from '../MovieCard';
 
 import './HeaderContainer.scss';
 
-const HeaderContainer = ({ movieData, handleSearchValue, handleToggleSearchCriteria, searchOption, }) => {
-    return (
-        <div className="header-container">
-            <div className="buttons-container">
-                <a className="home-page-btn" href="."><b>netflix</b>roulette</a>
-                {movieData && <a className="home-page-btn" href=".">Search icon</a>}
-            </div>
-            {movieData ? <MovieCard movieData={movieData} /> :
-                <SearchContainer
-                    handleSearchValue={handleSearchValue}
-                    toggleSearch={handleToggleSearchCriteria}
-                    searchOption={searchOption} />}
-        </div>
-    );
-};
-HeaderContainer.defaultProps = {
+const DEFAULT_PROPS = {
```

Comment:
regular name convention is `defaultProps` and `propTypes`

Reply:
I'll change it in the next task